packages/shared/src/components/sidebar/DiscoverSection.tsx

```diff
@@ -17,11 +29,34 @@ export function DiscoverSection({
   enableSearch,
   ...defaultRenderSectionProps
 }: DiscoverSectionProps): ReactElement {
+  const seoSidebar = useFeature(feature.seoSidebar);
+  const isV1Sidebar = seoSidebar === SeoSidebarExperiment.V1;
   const { checkHasCompleted, completeAction, isActionsFetched } = useActions();
   const hasCompletedCommentFeed =
     !isActionsFetched || checkHasCompleted(ActionType.CommentFeed);
-  const discoverMenuItems: SidebarMenuItem[] = [
-    {
+  const locationPush = useCallback((path: string) => {
+    const isExtension = checkIsExtension();
+
+    if (!isExtension) {
+      return null;
+    }
+
+    return window.location.assign(`${webappUrl}${path}`);
+    // router is unstable
```

Comment:
Where is router here?

Reply:
Ah, was debugging something. Let me remove.